src/main/resources/assets/computercraft/lua/rom/apis/vector

```diff
@@ -76,9 +76,9 @@ local vmetatable = {
 
 function new( x, y, z )
 	local v = {
-		x = x or 0,
-		y = y or 0,
-		z = z or 0
+		x = type(x) == "number" and x or 0,
```

Comment:
Yeah, it is rather evil. I've had a prod through `rednet` and I can't see anything there which would cause similar issues.